src/blockchain/client/main.js

```diff
@@ -1,7 +1,11 @@
 // This script calls TwoPhaseCommitClient to demostrate a successful 2PC case.
-const blockchain_uri = 'ws://localhost:9545';
-const contract_addr = '0x007AEd8EDC5CCdc2b275c5E85284e01199695959';
-const shared_account = '0xf7533e520c218417628fc7d6f08bbef9239a39fd';
+const args = process.argv.slice(2);
+
+const blockchain_uri = 'ws://localhost:' + args[0];
+const contract_addr = args[1];
+// Don't change this since it's the first one in the deterministic ganache
+// client.
+const shared_account = '0x90F8bf6A479f320ead074411a4B0e7944Ea8c9C1';
```

Comment:
I've been using `truffle develop` instead of ganache which works fine so far - it gives me a different account list. I will update this to args[2] in later PR.